### ISSUE TITLE
Segmentation fault in conjunction with pytorch

### ISSUE THREAD

**Issue by AnselmC** [librosa/librosa#1044]
#### Description

Python segfaults when importing librosa before pytorch, but works fine if import statements are switched.

#### Steps/Code to Reproduce
```bash
parallels@anselm:GANNt-do-without-you$ python3
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import librosa
>>> import torch
Segmentation fault (core dumped)
parallels@anselm:GANNt-do-without-you$ python3
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> import librosa
>>> 
```

#### Expected Results

No seg fault regardless of import order

#### Actual Results

Segmentation fault when librosa is imported before torch, no seg fault when order is switched

#### Versions

```bash
Linux-4.15.0-72-generic-x86_64-with-Ubuntu-18.04-bionic
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0]
NumPy 1.17.4
SciPy 1.4.0
librosa 0.7.1
pytorch 1.3.0
```

**Comment by bmcfee** [comment]
That's real strange! I suspect if anything it's a problem in pytorch, since that's the import line actually causing the segfault. It would be helpful to have a backtrace to see what's exactly going wrong here, but I generally suggest taking it to the pytorch folks.

**Comment by AnselmC** [comment]
ok, will do.
just for references here's debugging info and the resulting backtrace
```bash
>>> import librosa
[New Thread 0x7ffff1565700 (LWP 31576)]
[New Thread 0x7fffeed64700 (LWP 31577)]
[New Thread 0x7fffec563700 (LWP 31578)]
[Thread 0x7fffec563700 (LWP 31578) exited]
[Thread 0x7fffeed64700 (LWP 31577) exited]
[Thread 0x7ffff1565700 (LWP 31576) exited]
>>> import torch

Thread 1 "python3" received signal SIGSEGV, Segmentation fault.
0x00007fffcef2fcbd in pybind11::detail::make_new_python_type(pybind11::detail::type_record const&) ()
   from /home/parallels/.local/lib/python3.6/site-packages/torch/lib/libtorch_python.so
(gdb) backtrace
#0  0x00007fffcef2fcbd in pybind11::detail::make_new_python_type(pybind11::detail::type_record const&) ()
   from /home/parallels/.local/lib/python3.6/site-packages/torch/lib/libtorch_python.so
#1  0x00007fffcef39d46 in pybind11::detail::generic_type::initialize(pybind11::detail::type_record const&) ()
   from /home/parallels/.local/lib/python3.6/site-packages/torch/lib/libtorch_python.so
#2  0x00007fffcf0b7a1e in torch::onnx::initONNXBindings(_object*) ()
   from /home/parallels/.local/lib/python3.6/site-packages/torch/lib/libtorch_python.so
#3  0x00007fffceeafdd0 in initModule() () from /home/parallels/.local/lib/python3.6/site-packages/torch/lib/libtorch_python.so
#4  0x00000000005fba6f in _PyImport_LoadDynamicModuleWithSpec ()
#5  0x00000000005fbced in ?? ()
#6  0x00000000005678ee in PyCFunction_Call ()
#7  0x000000000051171e in _PyEval_EvalFrameDefault ()
#8  0x0000000000508245 in ?? ()
#9  0x000000000050a080 in ?? ()
#10 0x000000000050aa7d in ?? ()
#11 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#12 0x0000000000509d48 in ?? ()
#13 0x000000000050aa7d in ?? ()
#14 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#15 0x0000000000509d48 in ?? ()
#16 0x000000000050aa7d in ?? ()
#17 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#18 0x0000000000509d48 in ?? ()
#19 0x000000000050aa7d in ?? ()
#20 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#21 0x0000000000509d48 in ?? ()
#22 0x000000000050aa7d in ?? ()
#23 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#24 0x0000000000509455 in _PyFunction_FastCallDict ()
#25 0x00000000005a55a1 in _PyObject_FastCallDict ()
#26 0x00000000005a65de in _PyObject_CallMethodIdObjArgs ()
#27 0x00000000004f729d in PyImport_ImportModuleLevelObject ()
#28 0x000000000050e4f1 in _PyEval_EvalFrameDefault ()
#29 0x0000000000508245 in ?? ()
#30 0x00000000005167b9 in ?? ()
#31 0x00000000005678ee in PyCFunction_Call ()
#32 0x000000000051171e in _PyEval_EvalFrameDefault ()
#33 0x0000000000508245 in ?? ()
#34 0x000000000050a080 in ?? ()
#35 0x000000000050aa7d in ?? ()
#36 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
---Type <return> to continue, or q <return> to quit---
#37 0x0000000000509d48 in ?? ()
#38 0x000000000050aa7d in ?? ()
#39 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#40 0x0000000000509d48 in ?? ()
#41 0x000000000050aa7d in ?? ()
#42 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#43 0x0000000000509d48 in ?? ()
#44 0x000000000050aa7d in ?? ()
#45 0x000000000050c5b9 in _PyEval_EvalFrameDefault ()
#46 0x0000000000509455 in _PyFunction_FastCallDict ()
#47 0x00000000005a55a1 in _PyObject_FastCallDict ()
#48 0x00000000005a65de in _PyObject_CallMethodIdObjArgs ()
#49 0x00000000004f729d in PyImport_ImportModuleLevelObject ()
#50 0x000000000050e4f1 in _PyEval_EvalFrameDefault ()
#51 0x0000000000508245 in ?? ()
#52 0x000000000050b403 in PyEval_EvalCode ()
#53 0x0000000000635222 in ?? ()
#54 0x00000000004ad8e5 in ?? ()
#55 0x00000000004afd04 in PyRun_InteractiveLoopFlags ()
#56 0x0000000000638c73 in PyRun_AnyFileExFlags ()
#57 0x0000000000639631 in Py_Main ()
#58 0x00000000004b0f40 in main ()
```

**Comment by bmcfee** [comment]
> just for references here's debugging info and the resulting backtrace

Cool -- definitely looks like a not-librosa problem.  However it might be triggered by an upstream dependency, eg joblib or numba.

**Comment by eun-sik** [comment]
I have same issue. I solved it by updating scipy version (1.4.1)
previous scipy version is 1.4.0

**Comment by bmcfee** [comment]
Since this is upstream of us, I'm closing this issue out.
